WcaOnRails/db/migrate/20161118141833_change_latitude_and_longitude_default.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+class ChangeLatitudeAndLongitudeDefault < ActiveRecord::Migration
+  def up
+    change_column_null(:Competitions, :latitude, true)
+    change_column_null(:Competitions, :longitude, true)
+    change_column_default(:Competitions, :latitude, default: nil)
+    change_column_default(:Competitions, :longitude, default: nil)
+
+    execute <<-SQL
+      UPDATE Competitions
+      SET latitude = NULL, longitude = NULL
+      WHERE (latitude = 0 AND longitude = 0)
+    SQL
+  end
+
+  def down
+    change_column_default(:Competitions, :latitude, 0)
+    change_column_default(:Competitions, :longitude, 0)
+    change_column_null(:Competitions, :latitude, false)
```

Comment:
What does this do to existing rows with NULL in these columns? I think this needs to use the optional 4 argument. See http://apidock.com/rails/ActiveRecord/ConnectionAdapters/SchemaStatements/change_column_null